src/index.js

```diff
@@ -93,20 +104,68 @@ export default class Scrl {
             // this.triggerEvent('scrollDone')
         }
     }
-
-    _update = () => {
+    
+    /**
+     * Updates and returns the distance the scroll has still to go
+     * @returns float
+     */
+    _getUpdatedDistance = () => {
         const distance = this._targetPositionYWithOffset - this._positionY;
         const attraction = distance * this.options.acceleration;
-
+        
         this._velocityY += attraction;
-
+        
         this._velocityY *= this.options.friction;
         this._positionY += this._velocityY;
-
+        
```

Comment:
A few added tabs here and there. Is that on purpose?

Reply:
Ups sorry, of course not. I will just apply the same prettier as in swup, ok? :)